### DISCUSSION TITLE
Error starting the real-world-vendure project

### DISCUSSION THREAD

**Discussion by FaustVs** [vendurehq/vendure#900] [Q&A] [answered]
Hi, new to Vendure so apologies in advance if this is mistake on my part.

I wanted to have a look at some example plugins, and I can across the real-world-vendure repo, which looked a great place to start. I've followed the instructions in the readme docs, however when I start the application with **yarn start** I get the following TypeORM error:

`QueryFailedError: SqliteError: FOREIGN KEY constraint failed`

I noted there was a bugfix related to a similar issue listed against the release notes for v1.0.1, however from what I can see I'm using that version.

**Comment by michaelbromley** [comment, marked as answer]
Thanks for the report. I can reproduce this error - I'll look into it.

**Reply by michaelbromley** [reply]
OK, so the fix in v1.0.1 you mentioned relates only to running migrations which use Vendure's `runMigrations` function. So the solution for you is to simply set `synchronize: false` in your vendure config and it should then work. Then, if you make any changes which affect the DB schema, you will need to manually generate and run a migration as described here: https://www.vendure.io/docs/developer-guide/migrations/#generate-a-migration

**Reply by FaustVs** [reply]
Worked like a charm, many thanks!
